planning/behavior_path_planner/src/utils/avoidance/utils.cpp

```diff
@@ -755,18 +755,36 @@ void filterTargetObjects(
       }
     }
 
-    // force avoidance for stopped vehicle
-    {
+    const auto stop_time_longer_than_threshold =
+      o.stop_time > parameters->threshold_time_force_avoidance_for_stopped_vehicle;
+
+    if (stop_time_longer_than_threshold && parameters->enable_force_avoidance_for_stopped_vehicle) {
+      // force avoidance for stopped vehicle
+      bool not_parked_object = true;
+
+      // check traffic light
       const auto to_traffic_light =
         utils::getDistanceToNextTrafficLight(object_pose, data.current_lanelets);
+      {
+        not_parked_object = to_traffic_light < parameters->object_ignore_distance_traffic_light;
+      }
 
-      o.to_stop_factor_distance = std::min(to_traffic_light, o.to_stop_factor_distance);
-    }
+      // check crosswalk
+      const auto & ego_pose = planner_data->self_odometry->pose.pose;
+      const auto to_crosswalk =
+        utils::getDistanceToCrosswalk(ego_pose, data.current_lanelets, *rh->getOverallGraphPtr()) -
+        o.longitudinal;
+      {
+        const auto object_to_crosswalk = to_crosswalk - o.longitudinal;
```

Comment:
This variable is not used.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/3767/commits/1565c628885aead3127af250c6dddb5713ead5e0